pyproject.toml

```diff
@@ -83,6 +83,27 @@ cov = [
   "cov-report",
 ]
 
+[tool.hatch.envs.lab]
+extra-dependencies = [
+	"jupyterlab",
+	"notebook",
+	"voila",
+	"ipywidgets",
+	# vim please
+	"jupyterlab-vim",
+
+	"beautifulsoup4",
+	# read parquet files
+	# "pyarrow",
```

Comment:
why comment this?

Reply:
We are not reading parquet yet but in case we switch to using those, this comes handy.